packages/esm-patient-test-results-app/src/filter/filter-set.tsx

```diff
@@ -22,43 +23,74 @@ const FilterSet: React.FC<FilterSetProps> = ({ hideFilterSetHeader = false }) =>
   const { t } = useTranslation();
   const { resetTree } = useContext(FilterContext);
   const [searchTerm, setSearchTerm] = useState('');
+  const [treeDataFiltered, setTreeDataFiltered] = useState(roots);
   const [showSearchInput, setShowSearchInput] = useState(false);
 
+  useEffect(() => {
```

Comment:
It seems like this should actually be calling `handleInputChange(searchTerm)`, right?